T2CC_APP/t2cc/app/src/main/java/com/example/t2cc/RegistrationActivity.java

```diff
@@ -247,6 +288,65 @@ private boolean validateRegistrationForm() {
     return valid;
   }
 
+  private boolean containLowerCase(String string){
+    boolean valid = false;
+    char ch;
+
+    for(int i=0; i < string.length(); i++) {
+      ch = string.charAt(i);
+
+      if(Character.isLowerCase(ch)) {
+        valid = true;
```

Comment:
I think we can break here, but we don't have to finish the loop. And same for the rest of the loops